src/messages/MClientRequest.h

```diff
@@ -51,7 +51,7 @@ class MClientRequest : public Message {
   static const int COMPAT_VERSION = 1;
 
 public:
-  struct ceph_mds_request_head head;
+  struct ceph_mds_request_head head{};
```

Comment:
This doesn't make sense, if it's not straight forward. What should be the default value for `union ceph_mds_request_args` ? I think better have a constructor inside `ceph_mds_request_head` and have default values based on the operation the variables are used for, to be explicit.